chia/full_node/full_node.py

```diff
@@ -1831,11 +1831,17 @@ async def add_block(
             "transaction_block": False,
             "k_size": block.reward_chain_block.proof_of_space.size,
             "header_hash": block.header_hash,
+            "fork_height": 0,
+            "rolled_back_records": 0,
```

Comment:
it looks like we're fine with omitting fields that aren't relevant. for example: `block_cost` and `block_fees`.
including `rolled_back_records` as 0 makes a certain amount of sense, but I'm not sure about including `fork_height` as 0 by default. if that value actually ends up in the result, it will look like an extremely deep reorg

Reply:
Updated this to `None` as default values instead of zero (since its a bit easier to parse that out vs a sometimes there sometimes not field in chia-exporter)